app/views/support_interface/regions/edit.html.erb

```diff
@@ -5,29 +5,30 @@
 <%= form_with model: [:support_interface, @region] do |f| %>
   <%= f.govuk_error_summary %>
 
+  <%= render "shared/support_interface/forms/application_form", f: %>
   <%= f.govuk_check_box :application_form_skip_work_history, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Skip work history" } %>
-  <%= f.govuk_check_box :reduced_evidence_accepted, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Accept reduced evidence" } %>
-
-  <%= f.govuk_collection_select :sanction_check, [
-    OpenStruct.new(id: 'online', name: 'Online'),
-    OpenStruct.new(id: 'written', name: 'Written'),
-    OpenStruct.new(id: 'none', name: 'None')
-  ], :id, :name, label: { text: "Sanction check" } %>
-
-  <%= f.govuk_collection_select :status_check, [
-    OpenStruct.new(id: 'online', name: 'Online'),
-    OpenStruct.new(id: 'written', name: 'Written'),
-    OpenStruct.new(id: 'none', name: 'None')
-  ], :id, :name, label: { text: "Status check" } %>
+  <%= render "shared/support_interface/forms/preliminary_check_form_fields", f: %>
+  <%= render "shared/support_interface/forms/eligibility_checker", f: %>
```

Comment:
Similarly the `eligibility_skip_questions` field doesn't exist on regions so we should remove it.